iree/jax/binary.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List
+from .program_api import Program
+
+try:
+  import iree.compiler.tools
+  import iree.runtime
+
+  _config_cache: Dict[str, iree.runtime.system_api.Config] = dict()
+  def get_rt_config(driver_name: str):
+    driver = _config_cache.get(driver_name)
+    if driver is None:
+      driver = iree.runtime.system_api.Config(driver_name)
+      _config_cache[driver_name] = driver
+    return driver
+
+except ImportError:
+  pass
+
+__slots__ = [
+    "Binary",
+]
+
+class Binary:
```

Comment:
(I keep hthinking "binary op" rather than executable binary)

Reply:
Done - changed the name to `iree`